components/skeleton/skeleton-mixin.js

```diff
@@ -152,10 +153,12 @@ export const SkeletonMixin = dedupeMixin(superclass => class extends RtlMixin(su
 	static get properties() {
 		return {
 			/**
-			 * Renders the input as a [skeleton loader](https://github.com/BrightspaceUI/core/tree/main/components/skeleton)
+			 * Render the component as a [skeleton loader](https://github.com/BrightspaceUI/core/tree/main/components/skeleton).
 			 * @type {boolean}
 			 */
-			skeleton: { reflect: true, type: Boolean  }
+			skeleton: { reflect: true, type: Boolean  },
+			_skeletonSetByParent: { state: true },
+			_skeletonSetExplicitly: { reflect: true },
```

Comment:
I don't think either of these need to be reactive state properties and can just be private class properties instead. Reactive state is only useful if you'd like changing their values to trigger a re-render, but it looks like they're just used to calculate the skeleton value.

`SkeletonTestNestedGroup` _does_ use `_skeletonSetExplicitly` during its render to pass the value to its switch... but hopefully it still works.

Reply:
Ah, good point. Yeah Will play around with this.